src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs

```diff
@@ -317,5 +334,37 @@ private static async Task WriteFiles(string fileName, string fileContent)
                 ColoredConsole.WriteLine($"{fileName} already exists. Skipped!");
             }
         }
+
+        private static bool ResolveManagedDependencies(WorkerRuntime workerRuntime, bool? managedDependenciesOption)
+        {
+            if (workerRuntime != Helpers.WorkerRuntime.powershell)
+            {
+                if (managedDependenciesOption.HasValue)
+                {
+                    throw new CliException("Managed dependencies is only supported for PowerShell.");
+                }
+
+                return false;
+            }
+
+            if (managedDependenciesOption.HasValue)
+            {
+                return managedDependenciesOption.Value;
+            }
+
+            ColoredConsole.Write("Would you like to install the Azure modules and have these automatically managed in Azure? ");
+            var answer = SelectionMenuHelper.DisplaySelectionWizard(WorkerRuntimeLanguageHelper.ManagedDependenciesInstallationOptions()).ToString();
```

Comment:
I think the `DisplaySelectionWizard` along with the `InstallManagedDependencies` Enum is a bit of an overkill for a `[y/n]` scenario. I think a generic helper like `bool SelectionHelper.Confirm(string question)` would be simpler?

Reply:
Great suggestion. I have added a helper Confirm in SelectionMenuHelper. The code looks like this:

```
public static bool Confirm(string question)
{
	if (string.IsNullOrEmpty(question))
	{
		throw new CliException("Question cannot be null or empty.");
	}

	List<string> options = new List<string>() { "Yes", "No" };
	ColoredConsole.Write(question);
	var answer = DisplaySelectionWizard(options);
	ColoredConsole.WriteLine(TitleColor(answer));

	return answer.Equals("Yes", StringComparison.OrdinalIgnoreCase);
}
```